README.md

```diff
@@ -1,14 +1,19 @@
 # [VDOM](https://github.com/nteract/vdom)
 
-## Why use VDOM?
+## Why use VDOM
 
 - Write **Declarative** Pythonic layouts.
-- Render the layout in **Jupyter** frontends (jupyterlab, nteract).
-- Serialize layout for rehydration in your web app.
+- Use user-friendly declarative statements to create headings, prose, images,
```

Comment:
I feel like this would read better as:

```
- Create headings, prose, images, and more common user interface items with user-friendly declarative statements
```